backend/logic/courses/courses.js

```diff
@@ -261,12 +263,30 @@ var updateCourse = async (course, newTitle, newTags) => {
   }
 };
 
+let phaseOutCourse = async courseCode => {
+  let connection = await mysql.getNewConnection();
+  try {
+    await format.verifyCourse(courseCode);
+    await connection.query(
+      "UPDATE courses SET phased_out = TRUE WHERE course_code = ?",
+      courseCode
+    );
+    return true;
+  } catch (error) {
+    console.error(error);
+    throw error;
```

Comment:
Not recommended to throw back SQL errors back at frontend since users will be able to see the structure of the database. Consider returning a custom message or masking the error instead.

Reply:
Youre right ill fix that thanks!